### PR TITLE
Kill d2l-sequence-viewer

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**
I'm open to arguments for whether or not this warrants a major version bump. I personally think so, but I can be swayed.

Once this is merged I'll make a BSI PR to use the new `sequences` `d2l-sequence-viewer` component.

(I've also tested with BSI locally to make sure things are a-ok)

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Team Phoenix Members
+
+* @ChrisKraljevicD2L @ianlaidlaw @sarahboyd @pbrown-d2l
```
Comment (ChrisKraljevicD2L) [inline CODEOWNERS:3]: I'm not sure when our `CODEOWNERS` file should be expanded but this is copied from `d2l-sequence-viewer` for now.

File: demo/d2l-sequence-viewer/data/dummy [hunk 2]
```diff
@@ -0,0 +1 @@
+dummy text
```
Comment (ianlaidlaw) [inline demo/d2l-sequence-viewer/data/dummy:1]: what is this for?

File: CODEOWNERS (hunk 1 above)
Comment (apalaniuk) [reply CODEOWNERS:3]: You can also do `@Phoenix` and not have to manage these per-repo if that fits your needs.

**Comment by DiljotSG** [comment]
https://www.youtube.com/watch?v=i7gIpuIVE3k

File: demo/d2l-sequence-viewer/data/dummy (hunk 2 above)
Comment (ChrisKraljevicD2L) [reply demo/d2l-sequence-viewer/data/dummy:1]: I have a feeling it was once for [this activity](https://github.com/BrightspaceHypermediaComponents/sequences/blob/ckraljevic/kill-sequence-viewer/demo/d2l-sequence-viewer/data/l1-m2-activity7.json#L44) but I don't see any `href`s that refer to the actual `dummy.txt` file. I'll nuke it.

**Comment by ChrisKraljevicD2L** [comment]
@ianlaidlaw I agree that directories can definitely be squished. Possibly as a followup :+1:

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Team Phoenix Members
+
+* @Phoenix
```
Comment (apalaniuk) [inline CODEOWNERS:3]: Think I flubbed that shooting from the hip, that should be `@Brightspace/Phoenix` since these are global identifiers

**Pull request merged**